src/components/calcite-pick-list-group/calcite-pick-list-group.tsx

```diff
@@ -13,7 +13,7 @@ export class CalcitePickListGroup {
   // --------------------------------------------------------------------------
 
   /**
-   * The title used for all nested pick-list rows
+   * The title used for all nested calcite-pick-list rows. Not to be confused with the heading of the entire component.
```

Comment:
I think the heading prop for the pick list itself was removed so this second sentence doesn't make sense anymore.